tools/test.py

```diff
@@ -128,6 +128,10 @@ def parse_args():
 def main():
     args = parse_args()
 
+    assert args.out or args.show,\
+        'Please choice an operation(Save or Show the results)\
+         by argument --out or --show'
```

Comment:
The backslash can be avoided by using parentheses.
```python
    assert (args.out or args.show,
            'Please specify at least one operation (save or show the results) '
            'with the argument "--out" or "--show"')
```

Reply:
I make some modify to avoid the assertion of flake8